pkg/controller/kubedirectorcluster/members.go

```diff
@@ -1283,11 +1312,13 @@ func queueNotify(
 	// FQDNs of the affected members.
 	op := ""
 	deltaFqdns := ""
+
+	// At the time this function is called, members in this list are
+	// marked as creating, ready, upgrading or config error. The fqdnsList function
+	// will appropriately skip the ones that are still creating, or the
+	// ones in other states that are just reboots.
+
```

Comment:
This comment should be put back where it previously was; it refers to the modifiedRole.membersByState[memberCreating] list.

Reply:
Fixed